pkg/controller/plan/kubevirt.go

```diff
@@ -242,7 +285,7 @@ func (r *KubeVirt) buildSecret(vmRef ref.Ref) (object *core.Secret, err error) {
 //
 // Generated name for kubevirt VM Import CR secret.
 func (r *KubeVirt) nameForSecret(vmRef ref.Ref) string {
-	uid := string(r.Plan.UID)
+	uid := string(r.Plan.Status.Migration.Active)
```

Comment:
Easier to humans to visually correlate.  Also, we create a new secret for each migration (run).